DNA Methylation introduction section (#956)
greenelab/deep-review: content/04.study.md

Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.

### DNA Methylation

#### Introduction 

> DNA Methylation (DNAm), is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 

Reviewer: ```suggestion
DNA methylation, is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
```
Didn't look like DNAm was ever used. I'd remove it unless it's going to be used elsewhere.

Author: agree

> This DNA-level epigenetic modification regulates gene transcription, is critical in development, and alterations to DNA methylation are well-established as contributing to pathophysiology of many diseases including cancers. 

Reviewer: ☝️ needs a reference - it's common knowledge in some places, but it'd be good to point folks to one or more topical papers (maybe even a few + a nice summary review if you have one) since some of our readers will be ML folks, not biologists.

My impression is also that DNA methylation can also be affected by processes that alter genome accessibility, including transcription, and so it can also provides a readout of a cell's past states. If this isn't totally off base, it's probably good to mention it here too. Doesn't the TP53 status thing fall into this category?

Author: added references here for the basics of DNA methylation biology and methylation in disease. The point about genome accessibility is true in that there is a very high correlation of DNA methylation and histone/chromatin measures for accessibility/inaccessibility, though the relationship with time and cell states has mostly been studied in the context of lineage commitment during development and there is not nearly enough data within specific cell types to trace past cell states yet.

> Studies of DNA methylation have also shed light on the processes involving initial differentiation of stem cells, aging and pathogenesis in response to environmental exposures. 

Reviewer: References would help here, again - over-reference the bio side to help ML people get to relevant literature quickly.

> Traditional analytic approaches to DNA methylation data often focus on estimating differential DNA methylation between groups or related with an outcome using linear mixed effects models. 

Reviewer: add a ref or two

> In addition, a growing application of DNA methylation measures is to infer cellular or subject phenotypes from samples and either examine the relation of these phenotypes with outcomes or disease states directly, and/or include them in models as covariates. 

Reviewer: another ref or two for examples

For example, inference of subject age using DNA methylation clock approaches are  established [@tag:Horvath2013], and are starting to be applied to test the relation of biological age with disease risk and outcomes [@tag:Kresovich2019]. 

> Importantly, reference DNA methylation profiles of leukocytes are available to infer immune cell type proportions with measures of DNA methylation in peripheral blood DNA [@tag:Houseman2012],[@tag:Salas2018], a novel approach to immunophenotyping that only requires. 

Reviewer: ```suggestion
Different cell types have different DNA methylation profiles.
Reference DNA methylation profiles of leukocytes are available, which makes it feasible to infer immune cell type proportions with measures of DNA methylation in peripheral blood DNA [@tag:Houseman2012],[@tag:Salas2018], a novel approach to immunophenotyping that only requires. 
```
☝️ missing word at the end of this sentence. I also added a short sentence ot introduce the idea that DNA methylation differs between cell types to help the computational reader along.

Cell type inference is important for adjusting for cell-type composition in epigenome-wide association studies [@tag:Teschendorff2017].

> While utilizing reference-based libraries demonstrates strong predictive value for immune cell type estimation, these methods severely restrict the amount of underlying biology that can be understood and correlated with disease manifestations and phenotypes. When a referenced-based library is not available for use, unsupervised methods are able to estimate these immune profiles [@tag:Titus2017]. 

Reviewer: ```suggestion
While utilizing reference-based libraries demonstrates strong predictive value for immune cell type estimation, these methods severely restrict the amount of underlying biology that can be understood and correlated with disease manifestations and phenotypes.
When a referenced-based library is not available for use, unsupervised methods are able to estimate these immune profiles [@tag:Titus2017]. 
```

> Methods that do not rely on these reference libraries [@tag:Houseman2016], are limited by being unable to fully capture the nonlinearity of the methylation data. 

Reviewer: ```suggestion
However, methods that do not rely on reference libraries [@tag:Houseman2016] are unable to fully capture the nonlinearity of methylation data. 
```
I think I understood this correctly. Check before committing the suggestion 😁

> There are many promising deep learning approaches that serve to expand the number of CpG sites whose DNA methylation state can be studied by capturing the complex interactions between different regions of DNA and extract the complete set of informative biologically relevant features. 

Reviewer: ```suggestion
Deep learning approaches have numerous potential applications for DNA methylation data.
Imputation methods that capture complex interactions between different regions of DNA can expand the number of CpG sites whose DNA methylation state can be studied, and ideally these methods can derive their own informative, biologically-relevant features.
```

> The main approaches focus on: 1) estimating regions of methylation status and imputing missing methylation values, 2) performing classification and regression tasks, and 3) understanding latent embeddings of methylation states from which to extract biologically meaningful features, infer interpolated disease states, and uncover relevant CpG sites for the above prediction tasks.

Reviewer: ```suggestion
The primary deep learning methods developed to date focus on: 1) estimating regions of methylation status and imputing missing methylation values, 2) performing classification and regression tasks, and 3) using the latent embeddings of methylation states to derive biologically meaningful features, infer interpolated disease states, and uncover CpG sites that aid the above prediction tasks.
```

Reviewer: See if this keeps the sentiment you were going for.

Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.

> ### DNA Methylation

Reviewer: ```suggestion
### DNA methylation
```

With the change coming in from @agitter ths will line up the headers.

Traditional analytic approaches to DNA methylation data often focus on estimating differential DNA methylation between groups or related with an outcome using linear mixed effects models, so-called epigenome-wide association studies (EWAS) [@tag:Laird2010], [@tag:Wilhelm-Benartzi2013], [@tag:Liu2013], [@tag:Teschendorff2017]. 
In addition, a growing application of DNA methylation measures is to infer cellular or subject phenotypes from samples and either examine the relation of these phenotypes with outcomes or disease states directly, and/or include them in models as covariates [@tag:Titus2017], [@tag:Salas2018_GR], [@tag:Zhang2019], [@tag:Horvath2014], [@tag:Quach2017]. 
For example, inference of subject age using DNA methylation clock approaches are  established [@tag:Horvath2013], and are starting to be applied to test the relation of biological age with disease risk and outcomes [@tag:Kresovich2019]. 
Different cell types have different DNA methylation profiles.

> Reference DNA methylation profiles of leukocytes are available, which makes it feasible to infer immune cell type proportions with measures of DNA methylation in peripheral blood DNA [@tag:Houseman2012],[@tag:Salas2018], a novel approach to immunophenotyping that only requires DNA. 

Reviewer: ```suggestion
A novel approach to immunophenotyping combines measurements with Reference DNA methylation profiles of leukocytes, which are available, to infer immune cell type proportions [@tag:Houseman2012],[@tag:Salas2018].
This strategy is particularly helpful when only DNA is available from a sample.
```

Cell type inference is important for adjusting for cell-type composition in epigenome-wide association studies [@tag:Teschendorff2017].

> While utilizing reference-based libraries demonstrates strong predictive value for immune cell type estimation and has broad utiltiy, cell composition mixtures per se, and methods to incorporate estimates of mixures pose important considerations on the interpretation of underlying biology associated with disease manifestations and phenotypes. 

Reviewer: ```suggestion
While reference-based libraries have strong predictive value for immune cell type estimation and has broad utility, cell composition mixtures per se, and methods to incorporate estimates of mixtures pose important considerations on the interpretation of underlying biology associated with disease manifestations and phenotypes. 
```

> When a referenced-based library is not available for use, methods that do not rely on these reference libraries, so called reference-free deconvolution [@tag:Houseman2016], are available to decompose singal purported to be contributed by cell types.

Reviewer: ```suggestion
When a referenced library is not available for use, methods that do not rely on these reference libraries, so called reference-free deconvolution [@tag:Houseman2016], are available to decompose signal purported to be contributed by cell types.
```
